tests/responses/ok-test.js

```diff
@@ -213,6 +222,168 @@ test('Does not return included data if "included = false"', function (t) {
   });
 });
 
+test('Support simple sorting', function (t) {
+  t.plan(4);
+
+  sails.request({
+    url   : '/user',
+    data  : {sort: 'firstName'},
+    method: 'GET'
+  }, function (err, res, body) {
+    if (err) {
+      t.fail(err);
+    }
+    try {
+      t.equal(res.statusCode, 200, 'HTTP status code is 200');
+      t.equal(res.headers['Content-Type'], 'application/vnd.api+json', 'Sends jsonapi mime type');
+      t.ok(validateJsonApi(body), 'Body is a valid JSON API');
+      t.deepEqual(
+        body.data.map(function (d) {
+          return d.id;
+        }),
+        ['2', '3', '1'],
+        'Data is in correct order'
+      );
+    } catch (err) {
+      t.fail(err);
+    }
+    t.end();
+  });
+});
+
+test('Support descending sorting', function (t) {
+  t.plan(4);
+
+  sails.request({
+    url   : '/user',
+    data  : {sort: '-firstName'},
+    method: 'GET'
+  }, function (err, res, body) {
+    if (err) {
+      t.fail(err);
+    }
+    try {
+      t.equal(res.statusCode, 200, 'HTTP status code is 200');
+      t.equal(res.headers['Content-Type'], 'application/vnd.api+json', 'Sends jsonapi mime type');
+      t.ok(validateJsonApi(body), 'Body is a valid JSON API');
+      t.deepEqual(
+        body.data.map(function (d) {
+          return d.id;
+        }),
+        ['1', '3', '2'],
+        'Data is in correct order'
+      );
+    } catch (err) {
+      t.fail(err);
+    }
+    t.end();
+  });
+});
+
+test('Support sorting by multiple fields', function (t) {
+  t.plan(4);
+
+  sails.request({
+    url   : '/user',
+    data  : {sort: 'lastName,firstName'},
+    method: 'GET'
+  }, function (err, res, body) {
+    if (err) {
+      t.fail(err);
+    }
+    try {
+      t.equal(res.statusCode, 200, 'HTTP status code is 200');
+      t.equal(res.headers['Content-Type'], 'application/vnd.api+json', 'Sends jsonapi mime type');
+      t.ok(validateJsonApi(body), 'Body is a valid JSON API');
+      t.deepEqual(
+        body.data.map(function (d) {
+          return d.id;
+        }),
+        ['2', '3', '1'],
+        'Data is in correct order'
+      );
+    } catch (err) {
+      t.fail(err);
+    }
+    t.end();
+  });
+});
+
+/*
+ * not supported yet
+ *
+test('Supports fetching relationships', function (t) {
+  t.plan(2);
+
+  sails.request({
+    url   : '/author/1/relationships/books',
+    method: 'GET'
+  }, function (err, res, body) {
+    if (err) {
+      t.fail(err);
+    }
+    try {
+      t.equal(res.statusCode, 200, 'HTTP status code is 200');
+      t.equal(res.headers['Content-Type'], 'application/vnd.api+json', 'Sends jsonapi mime type');
+      t.ok(validateJsonApi(body), 'Body is a valid JSON API');
+    } catch (err) {
+      t.fail(err);
+    }
+    t.end();
+  });
+});
+ */
```

Comment:
Instead of commenting this out, if it's something you're planning to support in a future PR, you can use [`test.skip`](https://github.com/substack/tape#testskipname-cb).